modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_option.jsp

```diff
@@ -131,5 +131,5 @@ long cpOptionId = cpOptionDisplayContext.getCPOptionId();
 </aui:form>
 
 <liferay-frontend:component
-	module="js/edit_cp_option"
+	module="js/edit_cp_option_and_value"
```

Comment:
@dsitu I'm not sure about this file name, can you review it and let me know if it is ok with Liferay pattern? It was renamed like so because the same file is used in two jsp..

cc. @garofalog